### DISCUSSION TITLE
Accessing Alpha Preview "Triggers" and "Functions" on locally ran supabase

### DISCUSSION THREAD

**Discussion by jamalmoir** [supabase/supabase#8764] [Questions]
I'm running supabase locally for development purposes, and I'd like to use Triggers and Functions (in order to create a record in another table when an auth.users record gets created).

However, in my Database tab, this section is missing. In fact, there seem to be quite a few things missing when compared to the app.supabase.com version.

How can I enable these?

**Comment by GaryAustin1** [comment]
The UI is the only thing not present and what is consider alpha.   Triggers and database functions are part of Postgres.   You would use the SQL window to create and manage them.   
 https://www.postgresql.org/docs/current/sql-createfunction.html
 https://www.postgresql.org/docs/current/sql-createtrigger.html

**Reply by jamalmoir** [reply]
Cheers Gary.

Yeah, I've been using plain SQL to do the same thing, but I was wondering if I can manage them in the same way as on the main site. I guess not!

Thanks again!
